karma.conf.saucelabs.js

```diff
@@ -24,6 +24,14 @@ var browsers = {
     browserName: 'MicrosoftEdge',
     platform: 'Windows 10',
     version: '13.10586'
+  },
+  sl_android: {
```

Comment:
Why do we need this, either we have both mobile test or we don't also Android 4.4 is very old. IMHO this is an overkill for testing since Saucelab is not that stable also